otherlibs/systhreads/st_stubs.c

```diff
@@ -19,6 +19,10 @@
 
 #define CAML_INTERNALS
 
+#define CAML_NAME_SPACE
```

Comment:
Any reason to prefer this approach over https://github.com/ocaml-flambda/ocaml-jst/commit/c39ae2e6b581e6a6e5649ea329e32a56ad8e2079? (Do we need the linked commit anywhere else?)

Reply:
Oh, this seems neater?  c39ae2e was a hack that might affect other things.

Reply:
I think it is slightly better because it silences
the deprecation error only in that file, rather
than for all the runtime files.